nameable.rb

```diff
@@ -0,0 +1,5 @@
+class Nameable
+  def correct_name
+    puts 'NotImplementedError'
+  end
```

Comment:
- [ ] Kindly make sure your `correct_name` method rise an error properly, you weren't asked to display it as a text the way you did here,

Hint: To raise an error in ruby, You may consider saying:
```rb
def correct_name
        raise NotImplementedError, "#{self.class} has not implemented method '#{__method__}'"
    end
```